app/views/products/_modal.html.haml

```diff
@@ -0,0 +1,17 @@
+#confirmPopup.modal.fade{"aria-hidden" => "true", "aria-labelledby" => "exampleModalLabel", tabindex: "-1"}
+  .modal-dialog
+    .modal-content
+      .modal-header
+        %h5#exampleModalLabel.modal-title Delete Products
+        %button.btn-close{"aria-label" => "Close", "data-bs-dismiss" => "modal", type: "button"}
+      .modal-body
+        - if many
```

Comment:
let's make this local name more detailed, more_products as example or smth like.